src/shared/crypto/bitcoin/bitcoin.utils.ts

```diff
@@ -61,36 +59,49 @@ export function decodeBitcoinTx(tx: string) {
   return btc.RawTx.decode(hexToBytes(tx));
 }
 
-const concat = P.concatBytes;
-
-function formatKey(hashed: Uint8Array, prefix: number[]): string {
-  return btc.base58check.encode(concat(Uint8Array.from(prefix), hashed));
-}
-
-function getAddressFromWshOutScript(script: Uint8Array, network: BtcSignerNetwork) {
-  return btc.programToWitness(0, script.slice(2), network);
-}
-
-function getAddressFromWpkhOutScript(script: Uint8Array, network: BtcSignerNetwork) {
-  return btc.programToWitness(0, script.slice(2), network);
-}
-
-function getAddressFromTrOutScript(script: Uint8Array, network: BtcSignerNetwork) {
-  return btc.programToWitness(1, script.slice(2), network);
-}
-
-export function getAddressFromOutScript(script: Uint8Array, network: BitcoinNetworkModes) {
-  const outScript = btc.OutScript.decode(script);
-  // This appears to be undefined at times?
-  const bitcoinNetwork = getBtcSignerLibNetworkConfigByMode(network);
-
-  if (outScript.type === 'wsh') return getAddressFromWshOutScript(script, bitcoinNetwork);
-  else if (outScript.type === 'wpkh') return getAddressFromWpkhOutScript(script, bitcoinNetwork);
-  else if (outScript.type === 'tr') return getAddressFromTrOutScript(script, bitcoinNetwork);
-  else if (outScript.type === 'pkh') return formatKey(script, [bitcoinNetwork?.pubKeyHash]);
-  else if (outScript.type === 'sh') return formatKey(script, [bitcoinNetwork?.scriptHash]);
-  logger.error(`Unknown address type=${outScript.type}`);
-  return '';
+// Outscript types (btc-signer lib):
```

Comment:
Maybe we can add these in a types file?

Reply:
Yep, I can do that as temp solution. I really need to put up an issue in the lib and ask him to export the types.

Reply:
make a PR on the repo to export this

Reply:
This was really left as a ref, do we really need the types here?